src/main/java/io/odpf/firehose/sink/objectstorage/message/MessageDeSerializer.java

```diff
@@ -16,15 +20,24 @@
 
     public Record deSerialize(Message message) throws DeserializerException {
         try {
-            DynamicMessage dynamicMessage = protoParser.parse(message.getLogMessage());
+            if (message.getLogMessage() == null || message.getLogMessage().length == 0) {
+                throw new EmptyMessageException();
+            }
+
+            DynamicMessage logMessage = protoParser.parse(message.getLogMessage());
```

Comment:
name of the field use dynamicMessage, logmessage is before deserialization

Reply:
yes, will rename